process/tac-member-R&Rs.md

```diff
@@ -0,0 +1,13 @@
+# TAC Member Roles & Responsibilities
+
+- Commit to abiding by the OpenSSF Code of Conduct
+- Provide ongoing, technology-neutral technical guidance and sponsorship to Technical Initiatives throughout the respective lifecycle stages
+- Establish and drive an overall Technical Vision aligned with the Foundation's MVSR for the community 
+- Regularly attend TAC meetings, participate in discussions, voting, and ad-hoc meetings as needed to support Technical Initiatives
+- Approve, establish, structure, organize, and archive Technical Initiatives and associated policies and procedures 
+- Provide sponsorship and guidance to new Technical Initiatives that petition to join or be created within OpenSSF
+- Participate in Technical Initiatives regularly to ensure alignment with the Technical Vision and MVSR of OpenSSF, and to identify any resource or funding requirements
+- Coordinate such other technical community matters related to the success of Technical Initiatives and the mission of the OpenSSF
```

Comment:
"Coordinate such other"?? Something doesn't seem right here, unfortunately I'm not sure what the intent is so I can't really suggest a correction. Just remove "such"? But who is the coordination meant to be with?

Reply:
Thanks for the catch, @lehors! I was mashing together the ideas of a few other folks and grammar lost this battle!